underfs/cosn/pom.xml

```diff
@@ -26,6 +26,7 @@
 
   <properties>
     <build.path>${project.parent.parent.basedir}/build</build.path>
+    <ufs.cosn.version>3.1.0-5.8.5</ufs.cosn.version>
```

Comment:
add a comment to say that this should be kept in sync with the build scripts in `dev/scripts/src/alluxio.org/build-distribution/cmd/common.go`

Reply:
done